hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/opcodes/SelfDestructSuite.java

```diff
@@ -198,27 +292,34 @@ final Stream<DynamicTest> selfDestructFailsWhenBeneficiaryHasReceiverSigRequired
                         .has(contractWith().balance(ONE_HBAR)));
     }
 
-    final Stream<DynamicTest> selfDestructViaCallLocalWithAccount999ResultsInLocalCallModificationPrecheckFailed() {
+    final Stream<DynamicTest> selfDestructViaCallLocalWithAccount999ResultsInLocalCallModificationPrecheckFailed(
+            @NonNull final String evmVersion) {
         return hapiTest(
+                overriding("contracts.evm.version", evmVersion),
                 contractCreate(SELF_DESTRUCT_CALLABLE_CONTRACT).balance(ONE_HBAR),
                 contractCallLocal(SELF_DESTRUCT_CALLABLE_CONTRACT, "destroyExplicitBeneficiary", mirrorAddrWith(999L))
                         .hasAnswerOnlyPrecheck(LOCAL_CALL_MODIFICATION_EXCEPTION));
     }
 
-    final Stream<DynamicTest> testSelfDestructForSystemAccounts() {
+    final Stream<DynamicTest> testSelfDestructForSystemAccounts(@NonNull final String evmVersion) {
         final AtomicLong deployer = new AtomicLong();
         final var nonExistingAccountsOps = createOpsArray(
                 nonExistingSystemAccounts,
                 SELF_DESTRUCT_CALLABLE_CONTRACT,
                 DESTROY_EXPLICIT_BENEFICIARY,
                 INVALID_SOLIDITY_ADDRESS);
-        final var existingAccountsOps = createOpsArray(
-                existingSystemAccounts, SELF_DESTRUCT_CALLABLE_CONTRACT, DESTROY_EXPLICIT_BENEFICIARY, SUCCESS);
-        final var opsArray = new HapiSpecOperation[nonExistingAccountsOps.length + existingAccountsOps.length];
+        //        final var existingAccountsOps = createOpsArray(
```

Comment:
Is this commented on purpose?

Reply:
Meant to remove it - unused in current (existing) test.